snap_ci/pinned.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Pin commands to install vagrant, ansible, serverspec
+# Pinning commands will ensure that they are ran in each snap-ci stage. This is
+# useful when you only want to re-run one stage.
+#
+#
```

Comment:
Nit: replacing these `#`'s with newlines would make this file a little easier to read IMHO.